src/containerapp/azext_containerapp/tests/latest/test_containerapp_java_component.py

```diff
@@ -126,6 +175,14 @@ def test_containerapp_java_component(self, resource_group):
                 JMESPathCheck('properties.scale.minReplicas', 1),
                 JMESPathCheck('properties.scale.maxReplicas', 1)
         ])
+        self.cmd("containerapp env java-component gateway-for-spring update -g {} -n {} --environment {}".format(resource_group, gateway_name, env_name, route_yaml_name_update), checks=[
+            JMESPathCheck('name', gateway_name),
+            JMESPathCheck('properties.componentType', "SpringCloudGateway"),
+            JMESPathCheck('properties.provisioningState', "Succeeded"),
+            JMESPathCheck('length(properties.springCloudGatewayRoutes)', 0),
+            JMESPathCheck('properties.scale.minReplicas', 1),
+            JMESPathCheck('properties.scale.maxReplicas', 1)
+        ])
```

Comment:
If execute command `containerapp env java-component gateway-for-spring update` for a java-component that contains springCloudGatewayRoutes, what will happen?

Reply:
Good point! added that kind of check